ads/_config.js

```diff
@@ -210,6 +210,10 @@ const adConfig = jsonConfiguration({
     preconnect: ['https://mads.at.atwola.com', 'https://aka-cdn.adtechus.com'],
   },
 
+  'adtelligent': {
+    preconnect: ['https://s.adtelligent.com'],
```

Comment:
please try to implement renderStart API.

Reply:
@lannka Hello, sorry it took a long time to go back, but it's done